connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -154,7 +154,7 @@ export async function githubUpsertIssueActivity(
     documentUrl: issue.url,
     timestampMs: lastUpdateTimestamp,
     tags: tags,
-    parents: [],
+    parents: [issueNumber.toString(), repoId.toString()],
```

Comment:
Same here, documentId should be visible otherwise we risk divergence?

Reply:
Actually maybe it's not documentid but at least should be a variable that is used to set the ressource internal Id ?